src/baseballGame/baseballGame.js

```diff
@@ -0,0 +1,58 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MSG, ERROR_MSG } from "../Messages";
+import { getComputerNum, getUserNum } from "./getValue";
+
+// 3. 컴퓨터와 유저의 숫자 비교
+export async function compareNum() {
+  const computer_num = getComputerNum();
+
+  let ball = 0;
+  let strike = 0;
+  while (strike !== 3) {
+    let user_num = await getUserNum();
+    user_num = user_num.split("").map(Number);
```

Comment:
이렇게도 작성해 볼 수 있을 것 같습니다!
```suggestion
    user_num = Array.from(user_num, Number);
```

Reply:
오 이런 방법도 있군요 참고해서 다음에 써먹어 보도록 하겠습니다 감사합니다 :) ㅎㅎ